src/bindings/python/src/openvino/frontend/tensorflow/utils.py

```diff
@@ -264,10 +266,19 @@ def tf_function(args):
 
         input_needs_packing = True
 
+    def are_shapes_defined(shape: Union[List, Dict]):
+        if shape is None:
+            return False
+
+        if isinstance(shape, list):
```

Comment:
What will be if the list is empty?

Reply:
Agreed, better to add a check. Normally we should not have an empty list argument here: if user specified input=[] then `normalize_inputs` will convert them to `None`
https://github.com/openvinotoolkit/openvino/blob/372009b6f02aa6300c616895bd45ce107c46f849/tools/ovc/openvino/tools/ovc/convert_impl.py#L356

But if something goes wrong and this function is called with empty list from somewhere else then there will be a problem because
```
In [4]: np.all([])
Out[4]: True
```
it will try to create input tensors but it should not do that for empty shapes. Therefore added check for emptiness as well.